### PR TITLE
build(deps): update resvg from 0.33.0 to 0.40.0

### PR DESCRIPTION
**Pull request by panekj**

### CODE REVIEW

File: src/views/svg.rs [hunk 1]
```diff
@@ -67,7 +67,7 @@ impl Widget for Svg {
     fn update(&mut self, cx: &mut crate::context::UpdateCx, state: Box<dyn std::any::Any>) {
         if let Ok(state) = state.downcast::<String>() {
             let text = &*state;
-            self.svg_tree = Tree::from_str(text, &usvg::Options::default()).ok();
+            self.svg_tree = Tree::from_str(text, &usvg::Options::default(), font_db).ok();
```
Comment (panekj) [inline src/views/svg.rs:70]: @dzhou121 `resvg` now requires `&fontdb::Database` as 3rd input for the SVG `Tree`, do you have any advice/tip how to provide that without duplicating database init?

Comment (panekj) [reply src/views/svg.rs:70]: `FONT_SYSTEM.db` is private unfortunately

Comment (dzhou121) [reply src/views/svg.rs:70]: It has `.db()`

https://github.com/lapce/cosmic-text/blob/main/src/font/system.rs#L88

Comment (panekj) [reply src/views/svg.rs:70]: ok, it exists but on main, not in `floem` branch, but also it exists in `floem` branch but only for `no_std`

Comment (dzhou121) [reply src/views/svg.rs:70]: Yeah looking at the wrong thing. We'll need to expose it then I guess.

Comment (panekj) [reply src/views/svg.rs:70]: (or syncing with upstream would be nice... )
> This branch is [125 commits behind](https://github.com/lapce/cosmic-text/compare/main...pop-os%3Acosmic-text%3Amain) `pop-os/cosmic-text:main`.

Comment (dzhou121) [reply src/views/svg.rs:70]: Yeah at some point :)

**Comment by panekj** [comment]
Closing in favour of #406

**Pull request closed without merging**